thirdparty/views.py

```diff
@@ -0,0 +1,54 @@
+from django.http import JsonResponse
+from GHJM import settings
+from django.views import View
+from accounts.models import CustomUser
+from django.views.decorators.http import require_http_methods
+import boto3, uuid
+# Create your views here.
+AWS_ACCESS_KEY = getattr(settings, 'AWS_ACCESS_KEY')
+AWS_SECRET_KEY = getattr(settings, 'AWS_SECRET_ACCESS_KEY')
+AWS_STORAGE_BUCKET_NAME = getattr(settings, 'AWS_STORAGE_BUCKET_NAME')
+AWS_BUCKET_ROOT_FOLDER_NAME = getattr(settings, 'AWS_BUCKET_ROOT_FOLDER_NAME')
+DEFAULT_PROFILE_URL = getattr(settings, 'DEFAULT_PROFILE_URL')
+
+
+@require_http_methods(['POST'])
+def receive_img(request):
+    img = request.FILES.get('profile_picture_url')
+    
+    if img != None:
+        uploader = ProfileUpload(img)
+        img_url = uploader.upload()
+    
+    else:
+        img_url = DEFAULT_PROFILE_URL
```

Comment:
```suggestion
    img_url = DEFAULT_PROFILE_URL
    
    if img != None:
        uploader = ProfileUpload(img)
        img_url = uploader.upload()
```

스타일 차이일 것 같은데  저는 이렇게 하는 거 선호해요.

Reply:
@onee-only 께서 제시해준 코드가 더 좋아보이네요~ 반영할게요~